api/test/api/test_utils.clj

```diff
@@ -34,3 +35,11 @@
 (defn load-test-document-plan [filename]
   (with-open [r (io/reader (format "test/resources/document_plans/%s.edn" filename))]
     (edn/read (PushbackReader. r))))
+
+(defn rebuild-sentence [tokens]
+  (->> (map (fn [{type :type text :text}]
+                (case (keyword type)
+                  :WORD (str " " text)
+                  :PUNCTUATION text)) tokens)
```

Comment:
Can move `tokens` as first param in threading operator `(->> tokens (map...) ...)`